docs/00-getting-started.md

```diff
@@ -50,14 +50,89 @@ default:
     ssh_priv_key: ~/.ssh/my-private-key
 ```
 
-To get the required keys for Azure Batch and Azure Storage, please follow the below instructions:
-
-1. Log into Azure
+0. Log into Azure
 If you do not already have an Azure account, go to [https://azure.microsoft.com/](https://azure.microsoft.com/) to get started for free today.
 
    Once you have one, simply log in and go to the [Azure Portal](https://portal.azure.com) to start creating your Azure Batch account and Azure Storage account.
 
-2. Create a Storage account
+
+#### Using AAD
+To get the required keys for your Azure Active Directory (AAD) Service Principal, Azure Batch Account and Azure Storage Account, please follow these instructions. Note that this is the reccomended path for use with AZTK, as some features require AAD and are disabled if using Shared Key authentication.
+
+1. Register an Azure Active Directory (AAD) Application
+
+- Navigate to Azure Active Direcotry by searching in "All Services". Click "Properties" and record the value in the "Directory ID" field. This is your __tenant ID__.
+
+![](./misc/AAD_1.png)
+
+- Navigate to App Registrations by searching in "All Services".
+
+![](./misc/AppRegistrations_1.png)
+
+- Click the "+ New application registration" option at the top left of the window. Fill in the necessary fields for the "Create" form.
```

Comment:
I would highlight that they need to use the "Web API" option (which is not at all obvious...)